packages/relay-offline/src/Environment.ts

```diff
@@ -46,7 +46,10 @@ export class Environment extends RelayEnvironment {
 
         const options: OfflineFirstOptions<Payload> = {
             execute: (offlineRecord: OfflineRecordCache<Payload>) => this.executeStoreOffline(network, offlineRecord),
-            onComplete: (options: { offlineRecord: OfflineRecordCache<Payload>; response: any }) => {
+            ...others,
+        };
+        if (onComplete) {
```

Comment:
Shouldn't exist a default onComplete/onDiscard? Looks like here will fail the onComplete/onDiscard operation when called into offline-first.

Seens like need to be:

```
const onCompleteDefault = () => true
const onDiscardDefault = () => true // or false?

const { onComplete = onCompleteDefault, onDiscard = onDiscardDefault, network, ...others } = offlineOptions;
```

Reply:
Offline first has its default values:
https://github.com/morrys/wora/blob/master/packages/offline-first/src/index.ts#L70